package.json

```diff
@@ -21,8 +21,6 @@
       "strapi": "cd backend && pnpm run strapi",
       "test": "pnpm run -r test",
       "transform-svg": "cd frontend && pnpm run transform-svg",
-      "cypress:open": "cd frontend && pnpm run cypress:open",
```

Comment:
Oooh shoot I didn't know we had these scripts in the root `package.json`. We should probably add these scripts for `playwright`

Reply:
Yes! I think we should add them back and also update the `Test` section in read me.
Let's do it in a follow-up issue 👍

Reply:
Created a follow-up issue: https://github.com/iFixit/react-commerce/issues/1253.